ground/gcs/src/plugins/config/stabilization.ui

```diff
@@ -498,7 +498,7 @@
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
-      <number>2</number>
+      <number>0</number>

```

Comment:
Should just do the same as https://github.com/d-ronin/dRonin/pull/1574/commits/1b762d8c40b91fd12a283279336aafcf35c23eba IMO.